install_dsmain.sh

```diff
@@ -25,7 +25,4 @@ git clone git@github.com:cropguard-ai/ds-main.git
 
 # Install the repo
 cd ds-main
-pip install . -v
-
-# TODO: add ds-main as a dependency in chat-langchain's pyproject.toml
```

Comment:
Why did we remove the TODO exactly?

Reply:
Good point, we need to create a task in JIRA to keep track of it. Altough this formating was providing from reading the file.